python/paddleserver/poetry.lock

```diff
@@ -2057,162 +2084,173 @@ files = [
-    {file = "paddlepaddle-2.4.2-cp36-cp36m-manylinux1_x86_64.whl", hash = "sha256:9007f95d730c2c4f1f923b3c6d8e4686ca9a4399891a04dac216b86986c94504"},
-    {file = "paddlepaddle-2.4.2-cp36-cp36m-win_amd64.whl", hash = "sha256:33f72b9df75a11ec27c91e582c66c838481bfd9356ad043d71eda095bb964e0c"},
-    {file = "paddlepaddle-2.4.2-cp37-cp37m-macosx_10_14_x86_64.whl", hash = "sha256:e7c94d42def30154707f99597bab04de2b6e3ce16e186b3627f12f9c99070594"},
-    {file = "paddlepaddle-2.4.2-cp37-cp37m-manylinux1_x86_64.whl", hash = "sha256:a6bc43bed7f8db15218008784a7fc24ad68bcd11d138a0eb4d812932e20df204"},
-    {file = "paddlepaddle-2.4.2-cp37-cp37m-win_amd64.whl", hash = "sha256:d9d9eb4b3278ac5e55e712bfc9be1a230a3980f7d48f0cff53fc34decc4ed2ac"},
-    {file = "paddlepaddle-2.4.2-cp38-cp38-macosx_10_14_x86_64.whl", hash = "sha256:8bdf9e7b147c50d5c1a4e3dabebf511cd790173712a3187803632f687e0e4f97"},
-    {file = "paddlepaddle-2.4.2-cp38-cp38-macosx_11_0_arm64.whl", hash = "sha256:15ff2ae41f5336a9d3cc47b0a2762c7d7b87c1d856ced2b249c740493b38a385"},
-    {file = "paddlepaddle-2.4.2-cp38-cp38-manylinux1_x86_64.whl", hash = "sha256:256b4291c0e2ecb100f08efb454076cfde52664f4c623e709c9057149ce72fb9"},
-    {file = "paddlepaddle-2.4.2-cp38-cp38-win_amd64.whl", hash = "sha256:826cb5485a3b3178c92713967f80978182842b701e0bc2b65330efa3ce6a2c5e"},
-    {file = "paddlepaddle-2.4.2-cp39-cp39-macosx_10_14_x86_64.whl", hash = "sha256:616fba7259b42ae867cbf72659214e32493f9784960085505d67dd2e2ee1994f"},
-    {file = "paddlepaddle-2.4.2-cp39-cp39-macosx_11_0_arm64.whl", hash = "sha256:9b1839073ec8529fd725b1a1f9d8deb022e609bbb9b8f637c35b30a18ce54120"},
-    {file = "paddlepaddle-2.4.2-cp39-cp39-manylinux1_x86_64.whl", hash = "sha256:11cb4bf1c653c997ad766dd6307ee9525cd340d6d1f159faefd3ead272adc4ab"},
-    {file = "paddlepaddle-2.4.2-cp39-cp39-win_amd64.whl", hash = "sha256:47edb44ccd7ca44e39ddb8a675f714d78970d862b966298f65eec7a88731d8eb"},
+    {file = "paddlepaddle-2.5.1-cp310-cp310-macosx_10_9_x86_64.whl", hash = "sha256:0a52c4525aef8638f8ef4e4b60ac228a47c8c5b9a69eed7ca9dd9768c9af5a7b"},
+    {file = "paddlepaddle-2.5.1-cp310-cp310-macosx_11_0_arm64.whl", hash = "sha256:3d140dd5250d2ce6a617eab28e4942cf73e1e326c9a023273b22267983d56b56"},
+    {file = "paddlepaddle-2.5.1-cp310-cp310-manylinux1_x86_64.whl", hash = "sha256:a171ada892b415bef9fa073c8590b26b050b31c6e73d0cc4229d64dd3bd02d7d"},
+    {file = "paddlepaddle-2.5.1-cp310-cp310-win_amd64.whl", hash = "sha256:99f7907f166b625bb7ee58a26c376ac4dd0bf7723108a79293469497874b0a7d"},
+    {file = "paddlepaddle-2.5.1-cp311-cp311-macosx_10_9_x86_64.whl", hash = "sha256:7a78e67ff57a7819b97691d8597c03b95499313ed630b4325a13ccaf42556d8f"},
+    {file = "paddlepaddle-2.5.1-cp311-cp311-macosx_11_0_arm64.whl", hash = "sha256:ef2d25f36ed34b792922208cb17e67a46160a56466e9f4f7cda4965d6cf3412e"},
+    {file = "paddlepaddle-2.5.1-cp311-cp311-manylinux1_x86_64.whl", hash = "sha256:2131251fab6ce384e9fa29d1660a3d17f6a3f73a9229393a6d0aa3bd62c23cb6"},
```

Comment:
@andyi2it it looks like paddlepaddle has cp311 support. or did you mean something else?

Reply:
from what i see here, nothing speaks against it and we could revert [a5afe2b](https://github.com/kserve/kserve/pull/3075/commits/a5afe2be02b7dd6fe734c180dcc06cdc0c70806f)

Reply:
paddle CI is green https://github.com/kserve/kserve/actions/runs/6743532414/job/18331606984?pr=3075